app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,67 @@
+class FeedbacksController < ApplicationController
+  before_action :set_feedback, only: %i[ show edit update destroy ]
+
+  # GET /feedbacks or /feedbacks.json
+  def index
+    @feedbacks = Feedback.all
+  end
+
+  # GET /feedbacks/1 or /feedbacks/1.json
+  def show
+  end
+
+  # GET /feedbacks/new
+  def new
+    @feedback = Feedback.new
+    @feedback = Feedback.order created_at: :desc
+  end
+
+  # GET /feedbacks/1/edit
+  def edit
+  end
+
+  # POST /feedbacks or /feedbacks.json
+  def create
+    @feedback = Feedback.new(feedback_params)
+
+      if @feedback.save
+        redirect_to new_feedback_path
+      else
+        @feedback = Feedback.order created_at: :desc
+        render :new
+      end
+  end
+
+  # PATCH/PUT /feedbacks/1 or /feedbacks/1.json
+  def update
+    respond_to do |format|
+      if @feedback.update(feedback_params)
+        format.html { redirect_to @feedback, notice: "Feedback was successfully updated." }
+        format.json { render :show, status: :ok, location: @feedback }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @feedback.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /feedbacks/1 or /feedbacks/1.json
+  def destroy
+    @feedback.destroy
+    respond_to do |format|
+      format.html { redirect_to feedbacks_url, notice: <%= t 'Feedback was successfully updated.' %>}
+      format.json { head :no_content }
+    end
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
+    def set_feedback
+      @feedback = Feedback.find_by(params[:id])
```

Comment:
```suggestion
      @feedback = Feedback.find_by(id: params[:id])
```

- check the condition if `@feedback` is `nil`